### DISCUSSION TITLE
Projétos

### DISCUSSION THREAD

**Discussion by brunoroddrigues** [community/community#115404] [Repositories] [answered]
### Select Topic Area

Question

### Body

import requests

def create_github_issue(title, body, repository_owner, repository_name, access_token):
    url = f"https://api.github.com/repos/{repository_owner}/{repository_name}/issues"
    headers = {"Authorization": f"token {access_token}"}
    payload = {"title": title, "body": body}
    response = requests.post(url, headers=headers, json=payload)
    
    if response.status_code == 201:
        print("Issue created successfully!")
        return response.json()["html_url"]
    else:
        print(f"Failed to create issue. Status code: {response.status_code}")
        return None

# Informe os detalhes da issue aqui
title = "Discussão sobre nova funcionalidade"
body = "Vamos discutir a viabilidade e os requisitos para a implementação da nova funcionalidade XYZ."
repository_owner = "nome_do_proprietario_do_repositorio"
repository_name = "nome_do_repositorio"
access_token = "seu_token_de_acesso"

# Criar a issue
issue_url = create_github_issue(title, body, repository_owner, repository_name, access_token)
if issue_url:
    print(f"Issue URL: {issue_url}")
else:
    print("Failed to create issue.")

**Comment by Akash1134** [comment, marked as answer]
Hi @brunoroddrigues , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
